source/server/hot_restart_impl.cc

```diff
@@ -195,20 +212,18 @@ int HotRestartImpl::duplicateParentListenSocket(const std::string& address) {
 }
 
 void HotRestartImpl::getParentStats(GetParentStatsInfo& info) {
-  // There exists a race condition during hot restart involving fetching parent stats. It looks like
-  // this:
-  // 1) There currently exist 2 Envoy processes (draining has not completed): P0 and P1.
-  // 2) New process (P2) comes up and passes the INITIALIZING check.
-  // 3) P2 proceeds to the parent admin shutdown phase.
-  // 4) This races with P1 fetching parent stats from P0.
-  // 5) Calling receiveTypedRpc() below picks up the wrong message.
+  // There exists a race condition during hot restart involving fetching parent stats. It looks
```

Comment:
same can we not reformat unless necessary?

Reply:
Again, clang-format.  I think I'm on the exact same version as in the docker image; has this file not been touched since clang-format-3.6 was used?

Reply:
all the files get checked, so some other mismatch maybe?